.github/workflows/build.yml

```diff
@@ -49,9 +56,49 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
+    needs: packages  # all packages for all platforms must be built first
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64, amd64]
+        hv: [xen, kvm]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: update linuxkit cache for runner arch so we can get desired images
+        uses: actions/cache@v3
+        with:
+          path: ~/.linuxkit/cache
+          key: linuxkit-amd64-${{ github.sha }}
+      - name: load images we need from linuxkit cache into docker
+        run: |
+          make cache-export-docker-load-all
+      - name: clear linuxkit cache so we can load for target arch
+        if: ${{ runner.arch != 'X64' }}
         run: |
-          make V=1 ROOTFS_VERSION="$VERSION" HV=kvm eve  # note that this already loads it into docker
+          rm -rf ~/.linuxkit
+      - name: update linuxkit cache for our arch
+        if: ${{ runner.arch != 'X64' }}
+        uses: actions/cache@v3
+        with:
+          path: ~/.linuxkit/cache
+          key: linuxkit-${{ matrix.arch }}-${{ github.sha }}
```

Comment:
It seems we will load cache twice (amd64 and arm64) for arm64, which allow us to have all packages in place. But should we load the same cache (amd64) twice for amd64?

Reply:
Should it be `if: ${{ matrix.arch != 'amd64' }}` or so?

Reply:
Yes, you saw it the same time I did. I got the logic wrong, but it is fixed now.